x/when.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * @name when
+ *
+ * @description
+ * Execute a function and return the result when a condition is true, otherwise return the original object
+ *
+ * ```javascript [playground]
+ * import when from 'https://unpkg.com/rubico/dist/x/when.es.js'
+ *
+ * const isEven = (num) => num % 2 === 0;
+
```

Comment:
the comment blocks should have asterisks the whole way through for consistency and cronist to parse